src/interface/src/styleguide/treatment-expander/treatment-expander.component.ts

```diff
@@ -0,0 +1,121 @@
+import { Component, Input } from '@angular/core';
+import { NgIf, NgFor, NgClass } from '@angular/common';
+import { MatIconModule } from '@angular/material/icon';
+import { MatExpansionModule } from '@angular/material/expansion';
+import {
+  TreatmentTypeIconComponent,
+  TreatmentIconColor,
+} from '../treatment-type-icon/treatment-type-icon.component';
+import { SequenceIconComponent } from '../sequence-icon/sequence-icon.component';
+
+export interface rxType {
+  name: string;
+  year: number;
+}
+
+//TODO: This should live in a central place and possibly map to something that's not a string?
+export type TreatmentType =
+  | 'No Treatment'
+  | 'Moderate thin & Biomass removal'
+  | 'Heavy thin & Biomass removal'
+  | 'Moderate thin & Pile burn'
+  | 'Heavy thin & Pile burn'
+  | 'Moderate mastication'
+  | 'Heavy mastication'
+  | 'Prescribed fire'
+  | 'Heavy thin & RX fire'
+  | 'Mastication & RX fire';
+
+/**
+ * Expander component
+ * A component to be used in the treatments panel to show treatment details
+ * NOTE: this expects *either* a treatmentType OR a sequenceNumber to determine appearance
+ */
+@Component({
+  selector: 'sg-treatment-expander',
+  standalone: true,
+  imports: [
+    MatExpansionModule,
+    MatIconModule,
+    NgClass,
+    NgIf,
+    NgFor,
+    SequenceIconComponent,
+    TreatmentTypeIconComponent,
+  ],
+  templateUrl: './treatment-expander.component.html',
+  styleUrl: './treatment-expander.component.scss',
+})
+export class TreatmentExpanderComponent {
+  /**
+   * Optional title text -- explicitly overrides the derived title
+   */
+  @Input() title: string | null = null;
+  /**
+   * A treatment type (Optional)
+   */
+  @Input() treatmentType: TreatmentType | null = null;
+  /**
+   * A treatment sequence number  (Optional)
+   */
+  @Input() sequenceNumber: number | null = null;
+  /**
+   * A number or ratio indicating stand count
+   */
+  @Input() standCount: string = '0';
+  /**
+   * Whether or not this is the selected expander
+   */
+  @Input() selected = false;
+  /**
+   * Total number of acres
+   */
+  @Input() totalAcres = 100;
+  /**
+   * A list of prescriptions {name: string, year: number}
+   */
+  @Input() rxDetails: rxType[] = [];
+  openState = false;
+
+  // Retaining colors here, so the icon component can remain as agnostic as possible
+  readonly treatmentIcons: Record<TreatmentType, TreatmentIconColor> = {
```

Comment:
hmm this is also kinda bad. We are hardcoding all the prescriptions here.
Should this be an input instead to the component?

Reply:
So, I think the TreatmentExpander needs to receive the TreatmentType as input, but should the TreatmentIcon also receive the TreatmentType as input?

We'll need to map the types to color and icon type somewhere -- I'm open to whatever you think is maintainable.